.gitignore

```diff
@@ -15,5 +15,7 @@ yarn.lock
 /artifacts
 /tests/e2e/screenshots
 /tests/e2e/videos
+/tests/e2e/fixtures/wrapped
+/tests/e2e/fixtures/unwrapped
 .lighthouseci
```

Comment:
let's not programmatically wrap? we can prepare the wrapped fixtures directly instead